nislmigrate/utility/installed_services_information.py

```diff
@@ -1,25 +1,20 @@
 import logging
+from typing import List
 
-from nislmigrate.logs.migration_error import MigrationError
+from nislmigrate.argument_handler import ArgumentHandler
+from nislmigrate.extensibility.migrator_plugin import MigratorPlugin
 
 
-def should_list_installed_services(argument_handler):
-    return argument_handler.is_list_installed_services_migration_flag_present()
+def list_installed_services(argument_handler: ArgumentHandler):
```

Comment:
Should this be a class?  I thought it was recommended practice to wrap Python APIs in classes.

Reply:
Yeah everything should be a class I think. Although previously none of these utilities were classes since they are static anyway. Fixed.